src/main/webapp/app/components/tabs/CurationToolsTab.tsx

```diff
@@ -12,6 +12,7 @@ import { CURRENT_REVIEWER } from 'app/config/constants/constants';
 import { GeneType } from 'app/shared/model/firebase/firebase.model';
 import { onValue, ref } from 'firebase/database';
 import { FB_COLLECTION } from 'app/config/constants/firebase';
+import { Unsubscribe } from 'firebase/auth';
```

Comment:
I think the type should come from `firebase/database` instead

Reply:
Makes sense. I make that mistake in too many places. 😄